src/screens/ReceiveModal.js

```diff
@@ -74,12 +63,19 @@ export default function ReceiveModal() {
   }, []);
 
   const checksummedAddress = useMemo(() => toChecksumAddress(accountAddress), [accountAddress]);
+  const { height: deviceHeight } = useDimensions();
+  const { top } = useSafeAreaInsets();
 
   return (
-    <Container backgroundColor="rgba(0,0,0,0.85)" testID="receive-modal">
-      <TouchableBackdrop onPress={goBack} />
-      <Handle />
-      <ColumnWithMargins align="center" margin={24}>
+    <SimpleSheet
+      testID="receive-modal"
+      backgroundColor={'rgba(0,0,0,0.85)'}
+      useAdditionalTopPadding={200}
```

Comment:
i'd make sure this doesn't make android look really weird

Reply:
If we need this prop for Android, probably should do

```
additionalTopPadding={IS_ANDROID ? StatusBar.currentHeight : false}
```

Reply:
i didn't know we had an android app

Reply:
lmao